packages/airnode-examples/integrations/coingecko-e2e/README.md

```diff
@@ -0,0 +1,7 @@
+# Coingecko-e2e example integration
```

Comment:
I am fine with using the example in our e2e tests, but I would still want this to be runnable by users. People might want to use it to test this by using two different chains. This example would be available in via the examples CLI so it should be usable on its own.

Reply:
I very much agree we should have an authorizer example integration (and in fact there is a task for this: #1318), but the contracts used in this test are contrived and unrelated to what an actual use case would involve. I think it is better to have this specifically for E2E testing of cross-chain authorizer functionality and build a separate integration that covers a real use case.

Reply:
Actually, for me the big feature here is the cross chain authorization and not the Authorizer contracts themselves.

Reply:
That's fair. This as a usable integration will be a decent departure from the other existing integrations (e.g. this will have multiple providers, multiple chains and different contracts deployed to each), but I'll work on it.

Reply:
Done!